src/reader/reader.ts

```diff
@@ -47,24 +49,153 @@ export function putContentIntoLines(contents: string): Query[] {
 }
 
 /**
- * 1. Split on new line
- * 2. Filter out any lines that start with a comment
- * 3. Rejoin the lines together as a single string.
+ * Function to escape any special characters in a string to allow
+ * it to be passed to build a regex.
+ * Based on https://stackoverflow.com/a/32212181/2144578
+ */
+function regexEscape(str: string): string {
+  str = str.replace(/\\/g, "\\\\");
+  str = str.replace(/\-/g, "\\-");
+  str = str.replace(/\//g, "\\/");
+  str = str.replace(/\./g, "\\.");
+  str = str.replace(/\^/g, "\\^");
+  str = str.replace(/\$/g, "\\$");
+  str = str.replace(/\*/g, "\\*");
+  str = str.replace(/\+/g, "\\+");
+  str = str.replace(/\-/g, "\\-");
+  str = str.replace(/\?/g, "\\?");
+  str = str.replace(/\(/g, "\\(");
+  str = str.replace(/\)/g, "\\(");
+  str = str.replace(/\[/g, "\\[");
+  str = str.replace(/\]/g, "\\]");
+  str = str.replace(/\{/g, "\\{");
+  str = str.replace(/\}/g, "\\}");
+  str = str.replace(/\|/g, "\\|");
```

Comment:
Sorry, this is a joke, right? First, you can chain calls like `str.replace(...).replace(...).replace(...)` instead of this ceremony of reassigning to one variable. Second, this is extremely inefficient and just silly way how to do it. The solution is as trivial as `str.replace(/([\\/.^$-])/g, '\\$1')` (add the rest of the chars yourself, just be careful about chars with special meaning in regexp group).

Reply:
This could certainly be improved. As could your tone. Will fix what I can on my side.

Reply:
Fixed.